src/service/invoice-service.ts

```diff
@@ -374,14 +396,19 @@ export default class InvoiceService {
       });
     }
 
-    base.description = update.description;
-    base.addressee = update.addressee;
+    if (update.description) base.description = update.description;
+    if (update.addressee) base.addressee = update.addressee;
+    if (update.street) base.addressee = update.street;
+    if (update.postalCode) base.postalCode = update.postalCode;
+    if (update.city) base.postalCode = update.city;
+    if (update.country) base.postalCode = update.country;
+    if (update.reference) base.reference = update.reference;
```

Comment:
These new attributes are not yet covered in a test case, so please do so (for example in `invoice-service.ts` line 541.

Reply:
fixed